assets/stylesheets/dashboard.scss

```
@@ -39,9 +39,12 @@ td div.progress {
     .progress-bar-passed, .progress-bar-softfailed, .progress-bar-unfinished {
         a { color: $color-result; }
     }
-    .progress-bar-failed, .progress-bar-skipped {
+    .progress-bar-failed{
         a { color: white; }
     }
+    .progress-bar-skipped {
+        a { color: rgb(0, 0, 0); }
+    }
```

No strong opinion but I'd use either color names or the `rgb` syntax consistently and I'd also avoid the nested blocks:

```suggestion
    .progress-bar-failed a { color: white; }
    .progress-bar-skipped a { color: black; }
```